pkg/db/errors.go

```diff
@@ -0,0 +1,10 @@
+package db
+
+import "errors"
+
+var (
+	ErrBookNotFound     = errors.New("book not found")
```

Comment:
These can be made const because these are the variables which will not be modified once they are created. For eg. ErrBookNotFound will always have the same value no matter which handler uses it.